apps/condo/domains/contact/schema/Contact.js

```diff
@@ -39,6 +39,10 @@ const Contact = new GQLListSchema('Contact', {
             isRequired: false,
         },
 
+        unitType: {
```

Comment:
Why not to assign it directly without destructiring?

Reply:
Done